testing/src/cypress.config.ts

```diff
@@ -24,14 +24,6 @@ export default defineConfig({
   e2e: {
     // We've imported your old cypress plugins here.
     // You may want to clean this up later by importing these.
-    baseUrl: "https://dev-aest-sims.apps.silver.devops.gov.bc.ca/",
-  },
-  env: {
-    token_url:
-      "https://test.oidc.gov.bc.ca/auth/realms/jxoe2o46/protocol/openid-connect/token",
-    update_institution_url: "/api/institutions/institution",
-    studentURL: "https://test-aest-sims.apps.silver.devops.gov.bc.ca/student/login",
-    username: "",
-    password: ""
-  },
+    // baseUrl: "https://dev-aest-sims.apps.silver.devops.gov.bc.ca/",
```

Comment:
Why did we keep a commented URL value here?

Reply:
That was a miss, let me update it.

Reply:
done